tests/static_lib_depend_static_static/lib_static0/lib_static0/libsrc/static0_archive.xc

```diff
@@ -0,0 +1,13 @@
+#include "static0.h"
+#include "static1.h"
+#include "print.h"
+void static0_archive()
+{
+#ifdef __XS3A__
+    printstrln("static0 archive xs3a");
+#else
+    printstrln("static0 archive xs2a");
+#endif
+    static1();
```

Comment:
There isn't a static1 function in lib_static1 - looks like that was copied from the (disabled) wrapped testcase? I've got this test working with this removed, but just want to check that was the intention.

Reply:
This should be a call to static1_archive();

Reply:
I think there is a missing file lib_static1/src/static1.xc

Reply:
Ive added the missing file and added a couple of more calls in the test two call between the two libs